File: packages/gyroscope/ios/Plugin/Classes/Results/IsAvailableResult.swift
import Capacitor

@objc public class IsAvailableResult: NSObject, Result {
    private let isAvailable: Bool

    init(isAvailable: Bool) {
        self.isAvailable = isAvailable
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["isAvailable"] = isAvailable
        return result as AnyObject
    }
}

File: packages/proximity-sensor/android/src/main/java/io/capawesome/capacitorjs/plugins/proximitysensor/classes/results/IsAvailableResult.java
package io.capawesome.capacitorjs.plugins.proximitysensor.classes.results;

import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.proximitysensor.classes.interfaces.Result;

public class IsAvailableResult implements Result {

    private final boolean available;

    public IsAvailableResult(boolean available) {
        this.available = available;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("available", available);
        return result;
    }
}
